specification/edge/resource-manager/Microsoft.Edge/sites/readme.md

```diff
@@ -0,0 +1,88 @@
+# edgesites
+
+> see https://aka.ms/autorest
+
+This is the AutoRest configuration file for edge sites.
+
+## Getting Started
+
+To build the SDKs for My API, simply install AutoRest via `npm` (`npm install -g autorest`) and then run:
+
+> `autorest readme.md`
+
+To see additional help and options, run:
+
+> `autorest --help`
+
+For other options on installation see [Installing AutoRest](https://aka.ms/autorest/install) on the AutoRest github page.
+
+---
+## Suppression
+
+``` yaml
+directive:
+  - suppress: OperationsAPIImplementation
+    from: sites.json
+    reason: RP is in PublicPreview and no SDK has been released yet. Microsoft.Edge RP consist of multiple resources which are owned/maintained by different teams, so we follow folder structure for Service Group (explained here https://github.com/Azure/azure-rest-api-specs-pr/tree/RPSaaSMaster?tab=readme-ov-file#folder-structure-for-service-group). We do have operations api exposed from common-location/folder (https://github.com/Azure/azure-rest-api-specs-pr/blob/RPSaaSMaster/specification/edge/resource-manager/Microsoft.Edge/edge/preview/2024-02-01-preview/operations.json#L46C5-L46C43) so every resource need not expose it separately. There has been open issue [Avocado] Support service group folder scenario azure-sdk-tools#6201 for the same.
+```
+
+## Configuration
+
+### Basic Information
+
+These are the global settings for the edgesites.
+
+```yaml
+openapi-type: arm
+openapi-subtype: rpaas
+tag: package-2024-02-01-preview
+```
+
+### Tag: package-2024-02-01-preview
+
+These settings apply only when `--tag=package-2024-02-01-preview` is specified on the command line.
+
+```yaml $(tag) == 'package-2024-02-01-preview'
+input-file:
+  - preview/2024-02-01-preview/sites.json
+```
+---
+
+# Code Generation
+
+## Swagger to SDK
+
+This section describes what SDK should be generated by the automatic system.
+This is not used by Autorest itself.
+
+```yaml $(swagger-to-sdk)
+swagger-to-sdk:
+  - repo: azure-sdk-for-python-track2
```

Comment:
```suggestion
  - repo: azure-sdk-for-python
```
Please update it for all the `readme.md` in this PR.

Reply:
Updated